### PR TITLE
maintainer: remove unused parts of pull request template.

### PR DESCRIPTION
**Pull request by KaiSzuttor**
Description of changes:
 - remove this :arrow_double_down: 

PR Checklist
------------
 - [ ] Tests?
   - [ ] Interface
   - [ ] Core 
 - [ ] Docs?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,11 +2,3 @@ Fixes #
 
 Description of changes:
  -
```
Comment (jngrad) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- 
```

**Review by jngrad — APPROVED** [review]

**Comment by RudolfWeeber** [comment]
It would be helpful for compiling the release notes, if a Changelog Message would be part of the template

**Comment by KaiSzuttor** [comment]
the issue is if the template is too long, nobody will use it. That's actually the reason for this PR.

**Comment by KaiSzuttor** [comment]
also this PR is not related to adding things to the template in the future. will merge for now.

**Comment by jngrad** [comment]
@RudolfWeeber core developers can write the changelog directly to the wiki, where we have draft release notes for [4.2.0](https://github.com/espressomd/espresso/wiki/es-420-release-notes) and [4.1.3](https://github.com/espressomd/espresso/wiki/es-413-release-notes)

**Pull request merged**